SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj

```diff
@@ -176,7 +181,8 @@
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos";
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = SWIFT_PACKAGE;
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = Default;
+				SWIFT_VERSION = 4.0;
```

Comment:
The issue is that it needs to still build in Xcode 8 for now. I don't think it would with either 4.0 or 3.2? Or would it?